vcu118-run.py

```diff
@@ -301,6 +315,12 @@ def load_and_start_kernel(*, gdb_cmd: Path, openocd_cmd: Path, bios_image: Path,
         args += ["-ex", "symbol-file " + shlex.quote(str(kernel_debug_file.absolute()))]
         args += ["-ex", "load " + shlex.quote(str(kernel_image.absolute()))]
     args += ["-ex", "load " + shlex.quote(str(Path(bios_image).absolute()))]
+    args += ["-ex", "set $entry_point = $pc"] # Record the entry point to the bios
+    for core in range(1, num_cores):
+        args += ["-ex", "thread {:d}".format(core + 1)] # switch to thread (core + 1) (GDB counts from 1)
+        args += ["-ex", "si 5"] # execute bootrom on every core
+        args += ["-ex", "set $pc=$entry_point"] # set every core to the start of the bios
+    args += ["-ex", "thread 1"] # switch back to core 0
```

Comment:
Could guard this whole thing with `if num_cores > 1:`, not sure if I like that idea or not